rdkit/Chem/MolDb/Loader_orig.py

```diff
@@ -70,27 +70,26 @@ def ProcessMol(mol, typeConversions, globalProps, nDone, nameProp='_Name', nameC
         pD[pn] = typeConversions[colTyp][1](pv)
       else:
         pD[pn] = pv
-  else:
-    pD = {}
+
   if redraw:
     AllChem.Compute2DCoords(m)
   if not skipSmiles:
-    row.append(Chem.MolToSmiles(mol, True))
+    row.append(Chem.MolToSmiles(mol))
   row.append(DbModule.binaryHolder(mol.ToBinary()))
   row.append(pD)
   return row
 
 
 def ConvertRows(rows, globalProps, defaultVal, skipSmiles):
+  #  globalProps is a python dictionary
+  PropsSize: int = len(globalProps)
```

Comment:
we don't use type hints in RDKit code, please remove this.

Reply:
Thank you for your review. I will revert back into original